haiku-apps/drive_encryption/drive_encryption-1.0.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="A tool to encrypt devices and file images"
+DESCRIPTION="DriveEncryption is a suite of programs to allow you to \
+encrypt devices and file images. The encryption is done via the AES algorithm, \
+and protected via a password.
+It can also use devices created with TrueCrypt (http://www.truecrypt.org/), \
+available for Windows and Linux, as long as you've created them to use the \
+AES encryption algorithm and a RIPEMD-160 hash algorithm protected password."
+HOMEPAGE="https://github.com/chirayudesai/driveencryption"
+COPYRIGHT="2007-2014 pinc Software"
+LICENSE="
+	TrueCrypt
+	MIT
+	"
+REVISION="1"
+CHECKSUM_SHA256=""
```

Comment:
I can't get a CHECKSUM for a commit that doesn't exist yet. As is stated in the commit, I need work from axel on the actual app before the recipe will be finished. I can move it below SOURCE_URI though.